src/_guides/libraries/c-interop.md

```diff
@@ -139,10 +143,12 @@ Once you understand the hello_world example, you should be ready to look at the
 How you bundle (or _package_ or _distribute_)
 a C library with your package or app and then load that library
 depends on your platform and the type of library.
-For details, see [Binding to native code via FFI.][binding]
+For details, see the following:
```

Comment:
I suggest we add something about how to bundle in pure Dart apps, specifically how to do it for an app compiled with `dart2native`

Reply:
Would you use dart2aot before 2.6?